runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1406,7 +1422,13 @@ private void process(
               computationId,
               key,
               workItem.getShardingKey(),
-              workItem.getWorkToken());
+              workItem.getWorkToken(),
+              () -> {
```

Comment:
how about a single fun that returns a Closable?
That will help ensure they are matched up

Reply:
Done (after next force push)